<launch>
    <arg name="port" default="/dev/ttyUSB0" />
	<arg name="publish_tf" default="false"/>
	
	<node name="robot_communication" pkg="nitrabot_communication" type="robot_communication" >
		<param name="port" value="$(arg port)" />
	</node>

	<node name="encoder_to_odom" pkg="nitrabot_communication" type="encoder_to_odom" >
		<param name="publish_tf" value="$(arg publish_tf)"/>
	</node>

</launch>
